public/docs/ts/latest/guide/architecture.jade

```diff
@@ -163,27 +160,23 @@ figure
 
 :marked
   A _component_ controls a patch of screen called a *view*.
-  The shell at the application root with navigation links, a list of heroes, a hero editor ...
-  they're all views controlled by components.
+  The shell at the application root with navigation links, a list of heroes, and a hero editor are all views controlled by components.
```

Comment:
This is better, but this sentence is still hard to parse. I think it's "the shell at the application root with navigation links" that makes it difficult, especially since it seems like it's "with ... a list of heroes". Consider rewriting and using a bulleted list to clarify the separation between items.